### DISCUSSION TITLE
Question concerning usage of Sequential model.

### DISCUSSION THREAD

**Discussion by errhernandez** [pyg-team/pytorch_geometric#3619] [Q&A] [answered]
Hello! I am using Sequential to construct my model by concatenating different types of layers. If I wanted to freeze parameters associated to different layers, or apply different learning rates to different layers, how can this be done? 

In pytorch one can use an ordered dict to name different layers in Sequential, but this appears not to be possible in PyG's version of Sequential. 

Thanks!

**Comment by errhernandez** [comment, marked as answer]
Ok, I think I now know; it is simple enough:
```python
for layer in model.children():
     for name, parameter in layer.named_parameters():
          'do whatever with them here, for example:'
           print( name, parametr.shape )
```
Sorry for asking trivialities; I'm still learning ;-) I hope this is useful to someone.

**Reply by rusty1s** [reply]
You are right that PyTorch supports an `OrderedDict` as input, which is something we could potentially add as well. However, your current workaround works as well :)
